content/Personal/a_year_at_mozilla.rst

```diff
@@ -0,0 +1,281 @@
+a full remotee was definitely the most impacting life change.
+
+.. image:: images/twitter-quote-remote.png
+
+From previous experencies as freelancer, I was aware of my inabilty to
+keep a discipline when working from home.
+
+Thanks to `Areski <https://twitter.com/areskib>`_ I `landed at Betahaus
+<http://www.betahaus.es/mathieu-leplatre/>`_,
+probably the best place to work from! This place is full of awesome people
+from all around the world, working in very distinct areas, from journalism
+to wood craving through cartoonists and photographs.
+
+Still, interacting on a daily basis with remote colleagues through video
+calls and IRC can sometimes be very frustrating. Especially when the Internet connection
+is flaky, when proprietary software suck badly on Linux (*I'm looking at you Vidyo*)
+or when a cacophonous horde of motorcycles hijacks your microphone.
+
+Timezones are also challenging :) Since Mozilla employees are spread around the world,
+it's not rare to extend the work day of a few hours in order to get in touch
+with colleagues! I can't complain, everything has been very flexible so far.
+
+Favorably, Mozilla offers us the possibility to travel and meet regularly. We make
+our best to organize small workshops several times per quarter, in addition to the
+gatherings of the whole company twice a year. Those are super fun and extremely
+productive!
+
+
+Maintaining former projects
+---------------------------
+
+As part of the transition, I had to withdraw from the several open-source projects
+`I was maintaining at Makina Corpus <http://makina-corpus.com/blog/metier/2013/geotrek-histoire-dun-projet-libre>`_.
+
+Obviously, giving up on something you had cherished for years is delicate. Emotions
+come in the way.
+
+Some projects were not receiving the love and attention I thought they deserved and that made
+me sad. But I realized I couldn't continue to watch them all and suffered from not
+being able to provide thorough answers on issues and questions.
+
+I begged for help or support several times along the year, and eventually I left them :|
+That's probably part of the open-source darwinism.
+
+By the way, Janis' `Jazzband project <https://jazzband.co>`_ is awesome! Think about it
+instead of using the Github organization of your current company...
+
+
+Chocolate factory
+-----------------
+
+Working for an organization that has a good image everywhere around the globe
+is quite rewarding of course (...the typical *«Oh really! I use it at home,
+Firefox is the best search engine to send emails on Facebook»*).
+
+I remember very well the feeling of pride I had when I first came to the Mozilla
+office in Paris. Impressed by its stairs covered with a giant fox logo, I felt like
+I wasn't up to the job!
+
+The welcoming workmates there helped lowering the pressure quickly.
+By the way, special thanks to the teams in charge of the different
+offices, they are amazing! It is even sometimes a bit embarassing to feel spoiled!
```

Comment:
OK, thinking about this more, I have two problems with this sentence:

1. Feeling spoiled itself isn't embarrassing -- it's embarrassing that we are *being* this spoiled.
2. I don't think I would say that the emotion I feel when spoiled is embarrassment. For me it's a different kind of feeling, like feeling unworthy or guilty or something else. There's a Chinese expression [不好意思](http://www.sinosplice.com/life/archives/2005/07/12/pei-sei) which I always liked for this but I never found a good equivalent in English. But maybe this is just me and you actually feel embarrassment in which case this is fine. Or maybe there's no better word for it and we just have to use "embarrassing".

Here are some alternatives that I think are better:

1. "Sometimes I even feel embarrassed to be so spoiled!" (Just move "feel" to "embarrassed", which resolves the first problem but not the second.)
2. "We're embarrassingly spoiled!" / "We're so spoiled, it's embarrassing!" (In this use, "embarrassing" sounds OK to me because it's less like literal embarrassment but an intensifier.)
3. "Being so spoiled makes me feel lucky/overwhelmingly grateful/uneasy/anxious/abashed/disconcerted!" / "It's humbling to be so spoiled!" (As above, but with other emotions.)